src/vinyldns/client.py

```diff
@@ -565,6 +565,40 @@ def list_record_sets(self, zone_id, start_from=None, max_items=None, record_name
         response, data = self.__make_request(url, u'GET', self.headers, **kwargs)
         return ListRecordSetsResponse.from_dict(data)
 
+    def search_record_sets(self, start_from=None, max_items=None, record_name_filter=None,
+                           record_type_filter=None, record_owner_group_filter=None, name_sort=None, **kwargs):
+        """
+        Retrieves a list of RecordSets globally in the VinylDNS database based on search criteria.
+        A minimum of two alpha-numeric characters is required.
+
+        :param start_from: the start key of the page
+        :param max_items: the page limit
+        :param record_name_filter: only returns record_sets whose names contain filter string
+        :param record_type_filter: only returns record_sets whose type is present in the given list
+        :param record_owner_group_filter: only returns record_sets belonging to the given owner
+        :param name_sort: sort the results as per given order
+        :return: the content of the response
+        """
+        args = []
+        if start_from:
```

Comment:
Prefer to be explicit in these checks. I realize the code above uses the same type coercion pattern, but I'd rather we didn't repeat that.
```suggestion
        if start_from is not None:
```
I'd like to see this applied to all of the "parameter has been provided" checks below.

Reply:
Thanks, @remerle. I add explicit `None` checks for all parameters.